java/example.conf

```diff
@@ -0,0 +1,32 @@
+# This is an example ray config file.
+# To use this file, copy it to your classpath and rename it to 'ray.conf'.
+
+# For all available config items, see 'java/runtime/src/main/resources/ray.default.conf'.
+# For config file format, see 'https://github.com/lightbend/config/blob/master/HOCON.md'.
+
+ray {
+
+  // Path of your Ray home dir,
+  // can be either an absolute path, or a relative path to current work dir.
+  home = "/path/to/your/ray/home"
+
+  // Worker's mode, available options are:
+  // `WORKER`: indicates that this worker is a normal worker.
+  // `DRIVER`: indicates that this worker is a driver.
+  worker.mode = DRIVER
+
+  // Run mode, available options are:
+  // `SINGLE_PROCESS`: Ray will be running in a single Java process.
+  // `SINGLE_BOX`: All ray processes will be running on one node.
+  // `CLUSTER`: Ray will be running in a cluster.
+  run-mode = SINGLE_BOX
```

Comment:
At a high level, is the `conf` file for the "cluster" or do you need one `conf` file for each application?

Why do you need to specify `SINGLE_BOX` versus `CLUSTER`?

Reply:
1) this file defines the default values, each application should provide their own file to fit their scenario.
2) this is due to a legacy issue. Because `SINGLE_BOX` and `CLUSTER` use different mechanism to manage functions. Once we finish #2881, `SINGLE_BOX` can be removed.